packages/cms-base/components/public/cms/element/CmsElementProductListing.vue

```diff
@@ -72,12 +103,53 @@ setInitialListing(
             class="p-4 border rounded-lg shadow-md hover:shadow-lg transition-shadow duration-200 ease-in-out"
           />
         </div>
-        <div class="mt-10">
+        <div
+          class="grid grid-cols-1 gap-4 md:gap-6 lg:gap-8 p-4 md:p-6 lg:p-8"
+          :class="{ 'lg:grid-cols-2': getTotalPagesCount > 1 }"
+        >
           <SwPagination
             :total="getTotalPagesCount"
             :current="Number(getCurrentPage)"
             @changePage="changePage"
           />
+          <div class="text-center place-self-center mt-2 lg:mt-0">
```

Comment:
maybe let's show it only when there is more than minimal limit on page?

For example there is https://frontends-demo-git-feat-filter-count-8b03c7-shopware-frontends.vercel.app/CMS/ with 4 products and it looks unnecessary as pagination is also not visible 
<img width="1226" alt="image" src="https://github.com/shopware/frontends/assets/13100280/f9a7838c-f347-4a39-9cfe-b1d770c68965">


or as alternative - show pagination with 1 page then, for consistency

Reply:
Pagination is now always shown also when there is only one page.